packages/keybr-textinput-events/lib/types.ts

```diff
@@ -21,4 +21,5 @@ export type KeyEvent = {
   readonly metaKey: boolean;
   readonly location: number;
   readonly repeat: boolean;
+  readonly getModifierState: (keyArg: string) => boolean;
```

Comment:
Again, as I explained above, this should be a static array of strings:

```
  /** The state of keyboard modifier keys. */
  readonly modifiers: readonly string[];
```

There are only two modifiers that we care about, "CapsLock" and "NumLock", so a string array would work just fine.